ksql-parser/src/main/java/io/confluent/ksql/parser/SqlFormatter.java

```diff
@@ -175,11 +175,13 @@ protected Void visitSelect(final Select node, final Integer indent) {
     @Override
     protected Void visitSingleColumn(final SingleColumn node, final Integer indent) {
       builder.append(ExpressionFormatterUtil.formatExpression(node.getExpression()));
-      builder.append(' ')
-                .append('"')
-                // for backwards compatibility, we always quote with `""` here
-                .append(node.getAlias().toString(FormatOptions.noEscape()))
-                .append('"');
+      if (node.getAlias().isPresent()) {
+        builder.append(' ')
+            .append('"')
+            // for backwards compatibility, we always quote with `""` here
+            .append(node.getAlias().get().toString(FormatOptions.noEscape()))
```

Comment:
While we're in here do you fancy  switching this to use: `FormatOptions.of(IdentifierUtil::needsQuotes)`?

If not, would you mind raising a github issue under milestone 6.0 to do this?